contracts/pcv/ETHPCVDepositAggregator.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./IPCVDepositAggregator.sol";
+import "./PCVDeposit.sol";
+import "../refs/CoreRef.sol";
+import "../external/Decimal.sol";
+import "./balancer/IRewardsAssetManager.sol";
+import "./PCVDepositAggregator.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import "../libs/UintArrayOps.sol";
+
+/// @title ETH PCV Deposit Aggregator
+/// @notice A smart contract that aggregates erc20-based PCV deposits and rebalances them according to set weights ]
+contract ETHPCVDepositAggregator is PCVDepositAggregator {
+
+    address constant private WETH_ADDRESS = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
```

Comment:
You should be able to remove this and just use the weth address from the constants library instead.